### ISSUE TITLE
Add precompiled binaries

### ISSUE THREAD

**Issue by JaceCearK1** [ipatix/agbplay#12]
Hey @ipatix!
I find it pretty hard to compile the project because of all the library dependencies, which partly themselves depend on other projects to compile.
Would you mind adding precompiled binaries of 'agbplay' to the "Releases" section?
That way Windows users like me don't have to go through the struggle/confusion of installing all these huge amounts of libraries.
Thanks in advance. :)

**Comment by ipatix** [comment]
I've thought about that previously. So far I've only managed to get it running with Cygwin and I've found it  relatively easy to compile after installing all the packages listed in the Readme. I could try to statically link agbplay (if the library licenses allow that, haven't checked yet) but you'd need Cygwin anyway.
I could also try to get a more native version with Mingw, but I haven't done that yet and I don't know how difficult that is. I'll check into this. Feel free to remind me incase I forget.
In both of these cases you'd still need a compatible Terminal app on Windows. I'm pretty sure the default command prompt won't work.

So, the question arises: How much does it actually help? You'd still have to get around a few nasty things.

As for Visual Studio Binaries: No, I've tried it, I've tried a lot and I couldn't get the terminal output to work properly with the required unicode characters (I guess there we're encoding issues with UTF-8).

Last but not least, I kinda have to admit that writing agbplay for Terminal was a really BAD decision in terms of compatibility because you won't be able to download and just run it.
